database/migrations/2018_08_31_210632_create_areas_table.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateAreasTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('areas', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('area_country');
+            $table->string('area_city');
```

Comment:
Maybe separate countries and cities to avoid typo's in the country names and useful for multiple thinks like meetups that do have a known country but not city.
Or for users wanting to subscribe to countries

Reply:
good one, will take care of this.